docs/source/formulations_example.rst

```diff
@@ -0,0 +1,648 @@
+    )
+    config = config.add_columns(
+        variable=AttributeByTemplate(
+            name="tastiness",
+            headers=["margarita", "tastiness"],
+            template=LinkByUID.from_entity(tastiness_template)
+        ),
+        columns=[MeanColumn(data_source="tastiness"), StdDevColumn(data_source="tastiness")]
+    )
+
+The ``name`` is only used to link a variable to columns.
+The ``headers`` are used to structure the table *and also create the names of the eventual descriptors*.
+The figure below shows how some of the columns may be rendered in a GEM Table.
+Because they both have the "margarita" heading, "blend time" and "tastiness" are grouped together.
+
+.. figure:: _static/GEM_Table_formulation_properties.png
+    :align: center
+
+    GEM Table "margarita properties" columns
+
+We now add a set of three variables for each mixing ingredient: one for the identifier, one for the quantity, and one for the labels.
+The code block below shows how we would add a set of columns for "simple syrup."
+This must be repeated once for each ingredient.
+
+.. code-block:: python
+
+    from citrine.gemtables.variables import IngredientIdentifierInOutput, IngredientQuantityInOutput, IngredientLabelsSetInOutput
+    from citrine.gemtables.columns import ConcatColumn
+
+    ing_name = "simple syrup"
+    mix_template_link = LinkByUID.from_entity(mix_template)
+    identity_var = IngredientIdentifierInOutput(
+        name=f"{ing_name} identifier",
+        headers=[ing_name, "Identifier"],
+        ingredient_name=ing_name,
+        process_templates=[mix_template_link],
+        scope=scope
+    )
+    quantity_var = IngredientQuantityInOutput(
+        name=f"{ing_name} mass",
+        headers=[ing_name, "Mass"],
+        process_templates=[mix_template_link],
+        ingredient_name=ing_name,
+        quantity_dimension=IngredientQuantityDimension.MASS,
+    )
+    labels_var = IngredientLabelsSetInOutput(
+        name=f"{ing_name} labels",
+        headers=[ing_name, "Labels"],
+        process_templates=[mix_template_link],
+        ingredient_name=ing_name
+    )
+    config = config.add_columns(
+        variable=identity_var, columns=[IdentityColumn(data_source=identity_var.name)]
+    )
+    config = config.add_columns(
+        variable=quantity_var, columns=[MeanColumn(data_source=quantity_var.name)],
+    )
+    config = config.add_columns(
+        variable=labels_var,
+        columns=[
+            ConcatColumn(
+                data_source=labels_var.name,
+                subcolumn=IdentityColumn(data_source=labels_var.name)
+            )
+        ]
+    )
+
+The figure below shows how these three columns may be rendered in a GEM Table.
+
+.. figure:: _static/GEM_Table_formulation_ingredient.png
+    :align: center
+
+    GEM Table "simple syrup ingredient" columns
+
+Lastly, we register the configuration and build the table.
+Note that this is a long-running process.
```

Comment:
SHOULD - "... is often a long-running..." Some demos only take a few seconds.

Reply:
```suggestion
Note that this can be a long-running process depending on how many cells are in the table.
```